### PR TITLE
Add VisualBERT demo notebook

### PR DESCRIPTION
**Pull request by gchhablani**
In continuation with #10534, this PR adds demo for VisualBERT model.
I am planning to base it on the `LXMERT` examples, hence the copy-paste of files for now.

### CODE REVIEW

**Comment by gchhablani** [comment]
I have updated the demo. Turns out I didn't need to change a lot from the LXMERT demo. I have used the same files, just replaced the tokenizer, the model and the labels that are being used.

Requesting @LysandreJik @patil-suraj to review.

File: examples/research_projects/visual_bert/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# VisualBERT Demo
```
Comment (patil-suraj) [inline examples/research_projects/visual_bert/README.md:1]: Maybe mention here that the demo is for visual question answering.

**Review by patil-suraj — APPROVED** [review]
Thanks a lot for adding this example!

**Review by LysandreJik — APPROVED** [review]
Very nice, great work @gchhablani!

Also ping @eltoto1219 who contributed LXMERT and who might be interested in this :)

**Pull request merged**